distributions/axoflow-otel-collector/README_windows.md

```diff
@@ -0,0 +1,112 @@
+# Windows installation
+
+## Installation steps
+
+1. Download the proper artifact for your platform. (We provide MSI installers, as well as binary releases for amd64 and arm64 architectures.)
+
+2. Run the installer.
+
+The installer ships the following components:
+
+  * Installs the application (by default) to `C:\Program Files\Axoflow\OpenTelemetry Collector\axoflow-otel-collector.exe`
+  * A default configuration that must be edited before use.
+
+## Configuration
+
+Provide the OTLP connector's address and port. (E.g., `10.0.2.2:4317`)
+
+```yaml
+exporters:
+  otlp/axorouter:
+    endpoint: 10.0.2.2:4317
+    tls:
+      insecure: true
+```
+
+The default configuration supports the following formats.
+
+### Event log
+
+The default configuration collects from the following channels: `application`, `system`, `security`.
+
+**NOTE: If you would like to include additonal channels you can use the following snippet and add it as a new receiver. (Don't forget to include the additonal receiver in the pipeline of your choice).**
+
+```yaml
+receivers:
+  windowseventlog/<CHANNEL_NAME>:
+    channel: <CHANNEL_NAME>
+    raw: true
+```
+
+### DNS log
+
+You can retrieve the DNS logfile path with the following PowerShell command:
+
+```powershell
+(Get-DnsServerDiagnostics).LogFilePath
+
+C:\dns.log
+```
+
+Please note that you have to escape your backslashes in your path. (E.g., `C:\\dns.log`)
+
+```yaml
+receivers:
+  filelog/windows_dns_debug_log:
+    include: ['<ESCAPED_DNS_LOGFILE_PATH>']
+    resource:
+      com.axoflow.vendor: microsoft
+      com.axoflow.product: windows:dns
+    attributes:
+      win_log_type: dns_debug_log
+    start_at: beginning
+    multiline:
+      line_start_pattern: '\d\d?/\d\d?/\d{4}'
```

Comment:
I would suggest to leave this off, as this config is already outdated and can get outdated again quite frequently
```suggestion
```